Installing own starterkit fails

I am developing a custom starterkit. The package also has a custom namespace: `@ringods/starterkit-zoho-sites`.

Since I do not have a finished version yet, I added the starterkit like this in `devDependencies`:
```
    "@ringods/starterkit-zoho-sites": "file:../starterkit-zoho-sites",
```

At NPM level, the package is installed, but then I try to trigger the install of the starterkit in Patternlab:

```
$ npx @pattern-lab/cli install --starterkits @ringods/starterkit-zoho-sites
⠇ ⊙ patternlab → Installing starterkit: @ringods/starterkit-zoho-sites⊙ patternlab → fetchPackage: Fetching required dependencies from npm failed for @ringods/starterkit-zoho-sites with Error: Command failed: npm install pattern-lab/@ringods/starterkit-zoho-sites
npm ERR! code ENOLOCAL
npm ERR! Could not install from "pattern-lab/@ringods/starterkit-zoho-sites" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ringods/.npm/_logs/2019-09-28T13_53_10_692Z-debug.log


(node:74278) UnhandledPromiseRejectionWarning: Error: Command failed: npm install pattern-lab/@ringods/starterkit-zoho-sites
npm ERR! code ENOLOCAL
npm ERR! Could not install from "pattern-lab/@ringods/starterkit-zoho-sites" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ringods/.npm/_logs/2019-09-28T13_53_10_692Z-debug.log


    at makeError (/Users/ringods/.npm/_npx/74278/lib/node_modules/@pattern-lab/cli/node_modules/execa/index.js:172:9)
    at Promise.all.then.arr (/Users/ringods/.npm/_npx/74278/lib/node_modules/@pattern-lab/cli/node_modules/execa/index.js:277:16)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:74278) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:74278) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
⠏ ⊙ patternlab → Installing starterkit: @ringods/starterkit-zoho-sites
```

My package name got prefixed with `pattern-lab/` and I could trace that to this line in the code: 

https://github.com/pattern-lab/patternlab-node/blob/aea1c7f0d816a11cf9116637e65421c5f3f8804e/packages/cli/bin/install-starterkit.js#L15

Why is this done? Can this be changed so that any starterkit can be installed?
Or can/should I change the NPM namespace to `@pattern-lab`?

--- a/packages/cli/bin/install-starterkit.js
+++ b/packages/cli/bin/install-starterkit.js
@@ -12,8 +12,7 @@ const installStarterkit = (starterkit, config) =>
   wrapAsync(function*() {
     const sourceDir = config.paths.source.root;
     const name = starterkit.value || starterkit;
-    const url = name.startsWith('@pattern-lab/') ? name : `pattern-lab/${name}`;
-    yield checkAndInstallPackage(name, url);
+    yield checkAndInstallPackage(name);
     const kitPath = path.resolve('./node_modules', name);
     yield copyAsync(path.resolve(kitPath, 'dist'), path.resolve(sourceDir));
     let kitConfig;

--- a/packages/cli/bin/utils.js
+++ b/packages/cli/bin/utils.js
@@ -126,16 +126,15 @@ const copyWithPattern = (cwd, pattern, dest) =>
  * @func fetchPackage
  * @desc Fetches and saves packages from npm into node_modules and adds a reference in the package.json under dependencies
  * @param {string} packageName - The package name
- * @param {string} [url] - A URL which will be used to fetch the package from
  */
-const fetchPackage = (packageName, url) =>
+const fetchPackage = packageName =>
   wrapAsync(function*() {
     const useYarn = hasYarn();
     const pm = useYarn ? 'yarn' : 'npm';
     const installCmd = useYarn ? 'add' : 'install';
     try {
-      if (packageName || url) {
-        const cmd = yield spawn(pm, [installCmd, url || packageName]);
+      if (packageName) {
+        const cmd = yield spawn(pm, [installCmd, packageName]);
         error(cmd.stderr);
       }
     } catch (err) {
@@ -150,10 +149,9 @@ const fetchPackage = (packageName, url) =>
  * @func checkAndInstallPackage
  * Checks whether a package for a given packageName is installed locally. If package cannot be found, fetch and install it
  * @param {string} packageName - The package name
- * @param {string} [url] - A URL which will be used to fetch the package from
  * @return {boolean}
  */
-const checkAndInstallPackage = (packageName, url) =>
+const checkAndInstallPackage = packageName =>
   wrapAsync(function*() {
     try {
       require.resolve(packageName);
@@ -162,7 +160,7 @@ const checkAndInstallPackage = (packageName, url) =>
       debug(
         `checkAndInstallPackage: ${packageName} not installed. Fetching it now …`
       );
-      yield fetchPackage(packageName, url);
+      yield fetchPackage(packageName);
       return false;
     }
   });

